src/grandorgue/sound/GOSoundStream.cpp

```diff
@@ -319,7 +319,8 @@ void GOSoundStream::InitAlignedStream(
   /* Translate increment in case of differing sample rates */
   resample = existing_stream->resample;
   m_ResamplingPos.Init(
-    pSection->GetSampleRate() / existing_stream->audio_section->GetSampleRate(),
+    (float)pSection->GetSampleRate()
```

Comment:
@oleg68 I'm wondering how a change on left side of division will solve a problem of zero dividing?

Reply:
@larspalo thanks for the explanation!

Reply:
@larspalo The root cause was the attack sample of key noise had 48000, but the release sample had 44100. So when GO was switching to the release, existing_stream->audio_section->GetSampleRate() returned 48000 and pSection->GetSampleRate() returned 44100.

Maybe some other samples have 44100 for both attack and releases. It depends on certain samples. So if you doubt in the code correctness, please explain the details with the samples example.

Reply:
> The root cause was the attack sample of key noise had 48000, but the release sample had 44100.

Yes, the silent Stille.wav file used for the attack part of action releases (done in old style like before we introduced the HasIndependentRelease feature) is indeed at 48000 while the actual "release" sample is at 44100.

> So if you doubt in the code correctness, please explain the details with the samples example.

I would like to investigate the whole re-sampling handling further as it's very interesting, but right now I simply don't have time for it. As already said by someone else, the same type of re-sampling that we already have but with the "reverse" intention to add frequency modulation could be used for expanding the tremulant model and adding a wind model.